dev/camdenhells/REQUIREMENTS.md

```diff
@@ -173,7 +177,7 @@ Provide separate specification of a rendered view for a scene model. Each scene
   * Any number of frame renders to readable off-screen image buffers
 4. Multiple render targets per framebuffer (#1862)
   * E.g. for deferred lighting and ambient occlusion g-buffers
-  * All render targets are same size
+  * Different size attachments are supported per render target
```

Comment:
Do you mean that different render targets in a framebuffer can be backed by attached buffers of different sizes as long as they all are at least of the shared target size? I.e. "All render targets share the same render area size in their attached buffers"? 

Or do you mean there is a use case for different size render areas in simultaneously-attached color/depth-stencil buffers?

Reply:
There is a use case for different resolution attachments to a single render buffer. Sometimes you want a computer texture which due to what you are computing doesn't require full resolution, or on the flip side requires higher resolution.

Reply:
Scratch this. I don't know what I was thinking. The mechanism for multiple attachments can't possibly support this